src/Cli/dotnet/commands/dotnet-complete/CompleteCommandParser.cs

```diff
@@ -14,6 +14,12 @@ internal static class CompleteCommandParser
             HelpName = "command"
         };
 
+        public static readonly CliOption<bool> Detailed = new("--detailed")
+        {
+            Hidden = true,
+            Description = "Show detailed completion information"
```

Comment:
Localize?

Reply:
Good call 👍

Reply:
Actually that new option was vestigial as well - once we have completion script generation hooked up for users we can

a) add a new output format for `dotnet complete` that includes descriptions for the returned items, if any
b) update the shell providers that understand dynamic completions (currently bash and zsh) to use the new flag so that they can provide the descriptions and not just the values. this would be a nice UX improvement.